Ginger/GingerCoreCommonTest/TerminologyTests/GingerTerminologyUnitTests.cs

```diff
@@ -0,0 +1,457 @@
+            
+            //Act
+            string termValue = GingerTerminology.GetTerminologyValue(termResourceKey);
+
+            //Assert
+            Assert.AreEqual("Activities", termValue);
+        }
+
+        [TestMethod]
+        public void GingerGherkinTerm_TestActivities()
+        {
+            //Arrange
+            GingerTerminology.SET_TERMINOLOGY_TYPE = eTerminologyDicsType.Gherkin;
+            eTermResKey termResourceKey = eTermResKey.Activities;
+
+            //Act
+            string termValue = GingerTerminology.GetTerminologyValue(termResourceKey);
+
+            //Assert
+            Assert.AreEqual("Steps", termValue);
+        }
+
+        [TestMethod]
+        public void GingerTestingTerm_TestActivities()
+        {
+            //Arrange
+            GingerTerminology.SET_TERMINOLOGY_TYPE = eTerminologyDicsType.Testing;
+            eTermResKey termResourceKey = eTermResKey.Activities;
+
+            //Act
+            string termValue = GingerTerminology.GetTerminologyValue(termResourceKey);
+
+            //Assert
+            Assert.AreEqual("Steps", termValue);
+        }
+
+
+        [TestMethod]
+        public void GingerDefaultTerm_TestRunset()
+        {
+            //Arrange
+            GingerTerminology.SET_TERMINOLOGY_TYPE = eTerminologyDicsType.Default;
+            eTermResKey termResourceKey = eTermResKey.RunSet;
+
+            //Act
+            string termValue = GingerTerminology.GetTerminologyValue(termResourceKey);
+
+            //Assert
+            Assert.AreEqual("Run Set", termValue);
+        }
+
+        [TestMethod]
+        public void GingerGherkinTerm_TestRunset()
+        {
+            //Arrange
+            GingerTerminology.SET_TERMINOLOGY_TYPE = eTerminologyDicsType.Gherkin;
+            eTermResKey termResourceKey = eTermResKey.RunSet;
+
+            //Act
+            string termValue = GingerTerminology.GetTerminologyValue(termResourceKey);
+
+            //Assert
+            Assert.AreEqual("Run Set", termValue);
+        }
+
+        [TestMethod]
+        public void GingerTestingTerm_TestRunset()
+        {
+            //Arrange
+            GingerTerminology.SET_TERMINOLOGY_TYPE = eTerminologyDicsType.Testing;
+            eTermResKey termResourceKey = eTermResKey.RunSet;
+
+            //Act
+            string termValue = GingerTerminology.GetTerminologyValue(termResourceKey);
+
+            //Assert
+            Assert.AreEqual("Calendar", termValue);
+        }
+
+
+        [TestMethod]
+        public void GingerDefaultTerm_TestBusinessFlows()
+        {
+            //Arrange
+            GingerTerminology.SET_TERMINOLOGY_TYPE = eTerminologyDicsType.Default;
+            eTermResKey termResourceKey = eTermResKey.BusinessFlows;
+
+            //Act
+            string termValue = GingerTerminology.GetTerminologyValue(termResourceKey);
+
+            //Assert
+            Assert.AreEqual("Business Flow Features", termValue);
+        }
```

Comment:
in eTerminologyDicsType.Default you shouldn't have "Business Flow Features" it supposed to be "Business Flow"

Reply:
Fixed the TC's